engines/gob/detection/tables_adibou2.h

```diff
@@ -206,7 +212,8 @@
 	{
 		"adibou2",
 		"ADIBÙ 2.13 Ambiente",
-		AD_ENTRY1s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958),
+		AD_ENTRY2s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958,
+				"BECBD213.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```

Comment:
Same here. The italian version should be BECBI213.CD1

Reply:
can you provide me the md5 hash and filesize or is it the same?

Reply:
Same here too